src/ai/backend/manager/cli/__main__.py

```diff
@@ -305,10 +306,6 @@ async def _clear_terminated_sessions():
                 )
                 deleted_count = result.rowcount
 
-                vacuum_sql = "VACUUM FULL" if vacuum_full else "VACUUM"
-                log.info(f"Perfoming {vacuum_sql} operation...")
-                await conn.exec_driver_sql(vacuum_sql)
-
```

Comment:
Could you tell me why this block has been lifted?

Reply:
I moved the logic to the bottom function (_clear_old_error_logs) because I find it easier to identify the vacuum-related logic at the end rather than somewhere in the middle.

Reply:
@jopemachine  Since `VACCUM` operation itself does not depend on neither session clear nor error log clearing job I think it is better to separate it as an independent job package rather than appending it to prior jobs.